app/scripts/directives/ticketFinderContentTicket.dir.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+angular.module('yourStlCourts').directive('ticketFinderContentTicket', [function () {
```

Comment:
You don't need the `[]` around the function in this file. ng-annotate handles minification protection for us in this project.